src/openms/source/CHEMISTRY/AASequence.cpp

```diff
@@ -80,6 +82,70 @@ namespace OpenMS
     return *this;
   }
 
+    // Formulae that need to be added to the internal residues to get to fragment type
+    // Formulae from http://www.matrixscience.com/help/fragmentation_help.html
+    static const EmpiricalFormula& getInternalToFull()
+    {
+      static const EmpiricalFormula to_full = EmpiricalFormula("H2O");
+      return to_full;
```

Comment:
Thanks, addressed in recent commit. Thought about that, too, but the functions were not exactly the same so I had to rewrite more code.